### PR TITLE
Update Web Analytics documentation. (#230)

### LINKED ISSUE #230: Problem: Web Analytics page is outdated after release of G4 update
## Documentation version

* 2.7
* 2.8

## Relevant documentation pages

* https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/web-analytics/

## Description

First reported in the user forum, 2023-08-18: 

* https://groups.google.com/g/ica-atom-users/c/hPo77g8Af0k/m/8ftN2lDXBAAJ

At present, the Web Analytics documentation for Google Analytics and Google Tag Manager are still based on the older Universal Analytics (UA) model. This year Google updated its analytics to G4, introducing a number of significant changes that impact our existing documentation on this page. See the thread linked above for a good example. See also: 

* https://support.google.com/analytics/answer/10075209?hl=en&utm_id=ad

**ALSO: Old links to dead projects**

Additionally, as part of the conversation, it was pointed out that the Webalizer link on the page (pointing to a `.org` site) is incorrect, and should point to `.net` instead. In reviewing these alternatives, it seems they are all quite dated, and some more contemporary open source alternatives should be mentioned instead or in addition to these no-longer-maintained projects. 

## Recommended fix

An analyst will need to spend some time reviewing the G4 documentation, testing setting up both basic web analytics and (more importantly) Custom Dimensions with the new model, and then update the documentation and screenshots to match these changes. 

Additionally, I suggest the following link changes for the open source alternatives mentioned on the page: 

* Webalizer - change to: https://webalizer.net/

For **Log File Analysis** alternatives, consider adding: 

* GoAccess: https://goaccess.io/

For **Web Analytics** open source alternatives to Google, consider:

* Updating the current Piwik link (also incorrectly labeled as `Piwiki` currently) to Matomo (https://matomo.org), as the Piwik project has undergone a name change 
* Adding in Open Web Analytics (https://www.openwebanalytics.com/) as another alternative

### PR DESCRIPTION
**Pull request by melaniekung**

### CODE REVIEW

**Review by fiver-watson — APPROVED** [review]
Nice!

File: admin-manual/maintenance/web-analytics.rst [hunk 1]
```diff
@@ -223,31 +197,16 @@ Let's begin!
 Google Analytics web property setup
```
Comment (anvit) [inline admin-manual/maintenance/web-analytics.rst]: Do they still call it web property or should this section title be changed?

**Review by anvit — APPROVED** [review]
One small nitpick, but looks good otherwise! Great job Mel!

File: admin-manual/maintenance/web-analytics.rst (hunk 1 above)
Comment (melaniekung) [reply admin-manual/maintenance/web-analytics.rst]: good catch! thanks

**Pull request merged**